### PR TITLE
IRSR24-16 | @jdwjdwjdw | Update lists and teasers to align with h1

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
- [IRSR24-16](https://stanfordits.atlassian.net/browse/IRSR24-16): H1 left-aligned with cards
  - Achieved this by updating the max-width of teasers and lists
  - Works for layout `None` pages and layout `Full Width` pages.

# Review By (Date)
- When convenient

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Add a basic page with layout `None` and a basic page will layout `Full Width`. For each page, test out lists and teaser cards and confirm that they are now aligned with the page `h1`
3. Review code 🎃 

# Associated Issues and/or People
- [IRSR24-16](https://stanfordits.atlassian.net/browse/IRSR24-16): H1 left-aligned with cards

[IRSR24-16]: https://stanfordits.atlassian.net/browse/IRSR24-16?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by rebeccahongsf — APPROVED** [review]
LGTM 🚀

**Review by rebeccahongsf — CHANGES_REQUESTED** [review]
I was reviewing the code and realize that your styles were targeting `.layout--layout-paragraphs-one-column` for the full width. 

Is the goal to have the heading align with the start of the card/teaser in one column? 🤔 

When there are multiple items in the row, the first one aligns with the No Layout but doesn't align with the Full Width.

image

image

image

image

Edit: Added missing screenie for No Layout :D

**Comment by jdwjdwjdw** [comment]
@rebeccahongsf thanks for pointing that out! I made some adjustments to the two and three column layouts for full-width pages which should make things look more consistent. Let me know what you think.

File: src/scss/basic-page/full-width.scss [hunk 1]
```diff
@@ -7,3 +7,65 @@ $fa-font-path:  '~@fortawesome/fontawesome-free/webfonts';
...
+      max-width: calc(83.33333% - 6.66667px);
+    }
+
+    @include grid-media-max('lg') {
+      max-width: calc(83.33333% - 6px);
+    }
+
+    @include grid-media-max('sm') {
+      max-width: calc(83.33333% - 3.33333px);
+    }
```
Comment (rebeccahongsf) [inline src/scss/basic-page/full-width.scss]:
```suggestion
    @include responsive-flex-column(10);
```

File: src/scss/basic-page/full-width.scss [hunk 2]
```diff
@@ -7,3 +7,65 @@ $fa-font-path:  '~@fortawesome/fontawesome-free/webfonts';
...
+      max-width: calc(83.33333% - 6.66667px);
+    }
+  
+    @include grid-media-max('lg') {
+      max-width: calc(83.33333% - 6px);
+    }
+  
+    @include grid-media-max('sm') {
+      max-width: calc(83.33333% - 3.33333px);
+    }
```
Comment (rebeccahongsf) [inline src/scss/basic-page/full-width.scss]:
```suggestion
    @include responsive-flex-column(10);
```

**Review by rebeccahongsf — APPROVED** [review]
@jdwjdwjdw it's looking great now! Thank you for the speedy turnaround.

What you have totally works but I left a styling comment in case you wanted to use the [`responsive-flex-column()`](https://decanter.stanford.edu/docs/scss/index.html#mixin-mixin-responsive-flex-column) mixin instead of css. This mixin matches the `flex-10-of-12` class in one of the wrappers for the h1 😃

**Comment by jdwjdwjdw** [comment]
Thanks for bringing up that mixin @rebeccahongsf ! I added your suggestions. Makes ever thing much cleaner 🎉

**Pull request merged**